libs/server/repository/api-rest/src/lib/controllers/repository.controller.ts

```diff
@@ -25,7 +25,8 @@ export class RepositoryController {
   constructor(private repositoryFacade: RepositoryFacade) {}
 
   @Get()
-  list(@CurrentUserId() currentUserId: string): Promise<RepositoryEntity[]> {
+  @ApiOkResponse({ type: [ListRepositoriesReadModel] })
```

Comment:
```suggestion
  @ApiOkResponse({ type: ListRepositoriesReadModel })
```

Reply:
Arrays are brilliant as a return type of list :)